inst/www/flex_dashboard/flexdashboard.js

```diff
@@ -227,8 +227,12 @@ var FlexDashboard = (function () {
         li.attr('id', id);
       li.addClass('dropdown');
       // auto add "Share" title on mobile if necessary
-      if (!title && icon && (icon === "fa-share-alt") && isMobilePhone())
-        title = "Share";
+      if (!title && icon && (icon === "fa-share-alt")) {
+        if (isMobilePhone())
+          title = "Share";
+        else
+          title = '<span class="sr-only">Share</span>';
+      }
```

Comment:
@cpsievert above is my recommendation for the 'Share' button. On the desktop, it will be added as a hidden text using the bootstrap class `sr-only`.